packages/graphql_flutter/pubspec.yaml

```
@@ -28,6 +28,7 @@ dev_dependencies:
     sdk: flutter
   http: ^0.13.0
   test: ^1.17.12
+  melos: ^3.0.0
```

```suggestion
  melos: ^3.0.0
```

why do we need melos here?

I think melos should be only a dependency outside the package, in fact, we can fetch these dependencies with  `make dep`, but maybe I'm missing your reason here

---

Ah I found it

```
dart pub global run melos bootstrap
Found a melos.yaml file in "/home/runner/work/graphql-flutter/graphql-flutter" but no local installation of Melos.

From version 3.0.0, the melos package must be installed in a pubspec.yaml file next to the melos.yaml file.

For more information on migrating to version 3.0.0, see: https://melos.invertase.dev/guides/migrations#200-to-300

To migrate at a later time, ensure you have version 2.9.0 or below installed: dart pub global activate melos 2.9.0
make: *** [Makefile:10: dep] Error 1
```

---

Downgrading the global one